File: Java/Somatorio.java
public class Somatorio {
    public static int calcularSomatorio(int[] numeros) {
        int soma = 0;
        for (int numero : numeros) {
            soma += numero;
        }
        return soma;
    }

    public static void main(String[] args) {
        int[] numeros = {1, 2, 3, 4, 5}; // Substitua pelo conjunto de números que deseja somar
        int resultado = calcularSomatorio(numeros);
        System.out.println("O somatório é: " + resultado);
    }
}

File: Js/Somatorio.js
function calcularSomatorio(numeros) {
    let soma = 0;
    for (let i = 0; i < numeros.length; i++) {
        soma += numeros[i];
    }
    return soma;
}

const numeros = [1, 2, 3, 4, 5]; // Substitua pelo conjunto de números que deseja somar
const resultado = calcularSomatorio(numeros);
console.log("O somatório é: " + resultado);